en/docs/install-and-setup/setup/api-controller/managing-integrations-with-ctl.md

```diff
@@ -0,0 +1,1405 @@
+
+            -   Required :  
+                `--environment` or `-e` : Environment of the Micro Integrator to be searched
+            -   Optional :  
+                `--format` : pretty-print using templates  
+                `--pattern` or `-p` : Filter users by regex  
+                `--role` or `-r` : Filter users by role
+
+        !!! example
+            ```bash
+            apictl mi get users -e dev
+            ```
+            ```bash
+            apictl mi get users -r admin -e dev
+            ```
+            ```bash
+            apictl mi get users -p *tester* -e dev
+            ```
+
+    -   **Response**
+
+        ```go
+        USER ID
+        admin
+        capp-tester
+        ```
+
+2.  Get information of a specific user.
+
+    -   **Command**
+        ``` bash
+        apictl mi get users [user-name] -e <environment>
+        ```
+
+        !!! info
+            **Flags:**
+
+            -   Required :  
+                `--environment` or `-e` : Environment of the Micro Integrator to be searched
+            -   Optional :  
+                `--format` : pretty-print using templates
+
+        !!! example
+            ```bash
+            apictl mi get users capp-tester -e dev
+            ```
+
+    -   **Response**
+
+        ```go
+        Name - capp-tester
+        Is Admin - true
+        Roles - Internal/everyone, admin
+        ```
+
+### Add a new user
+
+You can use the below command to add a new user to a micro integrator.
+
+-   **Command**
+    ``` bash
+    apictl mi add user [user-name] -e <environment>
+    ```
+
+    !!! info
+        **Flags:**
+
+        -   Required :  
+            `--environment` or `-e` : Environment of the Micro Integrator to be searched
+
+    !!! example
+        ```bash
+        apictl mi add user capp-tester -e dev
+        ```
+
+-   **Response**
+
+    ```go
+    Adding new user [ capp-tester ] status: Added
+    ```
+
+### Delete a user
+
+You can use the below command to remove a user from the the Micro Integrator.
+
+-   **Command**
+    ``` bash
+    apictl mi delete user [user-name] -e <environment>
+    ```
+
+    !!! info
+        **Flags:**
+
+        -   Required :  
+            `--environment` or `-e` : Environment of the Micro Integrator to be searched
+
+    !!! example
+        ```bash
+        apictl mi delete user capp-tester -e dev
+        ```
+
+-   **Response**
+
+    ```go
+    Deleting user [ capp-tester ] status: Deleted
+    ```
+
+## Monitor Integration Artifacts
+
+Follow the instructions below to display a list of artifacts or get information about a specific artifact in an environment using CTL:
+
+1.  Make sure that the WSO2 Micro Integrator 4.0.0 version is started and that the 4.0.0 version of APTCTL is running.   
+    For more information, see [Download and Initialize the CTL Tool]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#download-and-initialize-the-ctl-tool).
+
```

Comment:
Remove above since we are going to mention these details above in the **Before you begin** section.

Reply:
Fixed with adb71b9